Makefile

```diff
@@ -49,7 +49,7 @@ CONTAINER_RUN := $(CONTAINER_CMD) "$(IMAGE)"
 
 GIT_COMMIT := $(shell git rev-parse HEAD 2> /dev/null || true)
 
-MANPAGES_MD = $(wildcard docs/*.md)
+MANPAGES_MD = $(wildcard docs/*.md) $(wildcard vendor/github.com/containers/image/docs/*.md)
```

Comment:
Interesting.  Hadn't thought about this, but this is a neat advantage to vendoring...